docs/docs/Integrations/Nvidia/integrations-nvidia-system-assist.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Integrate NVIDIA System-Assist with Langflow
+slug: /integrations-nvidia-system-assist
+---
+
+:::important
+This component is only available for Langflow users with NVIDIA GPUs on Windows systems.
+:::
+
+The **NVIDIA System-Assist** component enables interaction with NVIDIA GPU drivers through natural language prompts.
+
+For example, prompt System-Assist with `"What is my current GPU temperature?"` or `"Show me the available GPU memory"` to get information, and then tell System-Assist to modify your GPU settings.
+
+For more information, see the [NVIDIA G-assist project repository](https://github.com/NVIDIA/g-assist).
+
+## Prerequisites
+
+* Windows operating system
+* NVIDIA GPU
+* `gassist.rise` package installed (included with Langflow)
```

Comment:
Maybe it would be clear to a regular Langflow user, but I am confused. Does "included with Langflow" mean it's already installed or do I need to somehow enable it in Langflow?